### ISSUE TITLE
Roster and Movies buttons not functional in Preferences screen

### ISSUE THREAD

**Issue by peyre** [raceintospace/raceintospace#254]
I just noticed that in the Preferences screen (both at the start of the game and during the game), the 'Naut Roster button and the button to toggle between movies & stills don't do anything.  I don't consider this really crucial since I almost never use those buttons, but this might be something we should address at some point.

**Comment by rnyoakum** [comment]
These are intentionally disabled; the code for both of these has been torn out, and would need to be rewritten - I believe from scratch. Off the top of my head I don't think there are any remnants of either in the git repository (but I could be wrong).

**Comment by peyre** [comment]
Funny--they become functional once you start a game.  Maybe that bit of code in the new game screen wasn't eligible for open-sourcing, like the PBEM code.  Well, it's not an important item.

**Comment by peyre** [comment]
I just recompiled the game, and they seem to be working now.  Should we close this one?

**Comment by rnyoakum** [comment]
Um, I'm not sure what's going on, because they definitely don't do anything when I click them.

**Comment by peyre** [comment]
That is strange!  I just redownloaded and recompiled the game, and they work fine on my system.  Could the difference be that I'm running Linux?

**Comment by rnyoakum** [comment]
No, I'm running Linux. They don't work either during the New Game options or during the in-game Preferences menu. Maybe we're thinking about different things?

I pulled up the code in the RIS repository. For the Astronaut editing button, the code in prefs.cpp reads:
```
} else if ((x >= 146 && y >= 30 && x <= 219 && y <= 61 && mousebuttons > 0) || key == 'E') {
    // Edit astronauts has been ripped out

}
```
so it really shouldn't be doing anything. Similarly, the animation toggle code is:
```
} else if ((x >= 146 && y >= 70 && x <= 219 && y <= 101 && mousebuttons > 0) || key == 'A') {
    /* disable this option right now */
}
```
so it shouldn't do anything either.

**Comment by peyre** [comment]
Oh wait, we're (partly) talking about different things.  I haven't tried the custom roster thing, since I never use it.  I noticed that at one point, the buttons to toggle movies and sound on and off didn't work in the New Game screen.  That's no longer the case though; they seem to be working just fine.

**Comment by rnyoakum** [comment]
As of now, the [custom roster](https://github.com/raceintospace/raceintospace/issues/627) is functional again and the Audio toggle is [its own kind of janky](https://github.com/raceintospace/raceintospace/issues/402). When you (@peyre) last posted you said

> I noticed that at one point, the buttons to toggle movies and sound on and off didn't work in the New Game screen. That's no longer the case though; they seem to be working just fine.

For me, the option to switch between movies and still images is disabled; it isn't even a button. Is that the case for you, or are you seeing different behavior?

**Comment by peyre** [comment]
Yes, it's disabled for me too.  Maybe it was enabled at one point in the past.  I'm thinking it might be good to replace that with a button that takes you to a GUI for the config file, if we should implement that in the future (#338).

![NoButton](https://github.com/raceintospace/raceintospace/assets/3118014/be26a085-db88-4928-a08b-957f77f033f2)
